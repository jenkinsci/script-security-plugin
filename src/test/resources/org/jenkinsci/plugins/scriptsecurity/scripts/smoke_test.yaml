unclassified:
  scriptApproval:
    approvedSignatures:
    - method java.net.URI getHost

jenkins:
  systemMessage: Testing
  quietPeriod: 10
