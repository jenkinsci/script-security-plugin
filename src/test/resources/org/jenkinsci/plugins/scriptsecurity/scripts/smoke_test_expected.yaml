approvedSignatures:
- "method java.net.URI getHost"
